### PR TITLE
fix(files): remove unneeded extra place for the rows

### PR DESCRIPTION
**Pull request by JuliaKirschenheuter**
Fixes https://github.com/nextcloud/tables/issues/1232

## Screenshots

Before | After
---|---
Screenshot from 2024-07-23 09-37-49 | Screenshot from 2024-07-23 09-44-23

### CODE REVIEW

File: src/shared/components/ncTable/sections/CustomTable.vue [hunk 1]
```diff
@@ -464,7 +471,6 @@ export default {
 
 	thead {
 		position: sticky;
-		top: 118px;
```
Comment (juliushaertl) [inline src/shared/components/ncTable/sections/CustomTable.vue:467]: Adjusted in the other PR, let's remove this from here

File: src/shared/components/ncTable/sections/CustomTable.vue [hunk 2]
```diff
@@ -389,6 +389,13 @@ export default {
 	}
 }
 
+:deep(.text-editor__wrapper .paragraph-content) {
+	margin-bottom: 0!important;
+}
+:deep(.text-editor__wrapper div.ProseMirror > *:first-child) {
+	margin-top: 0!important;
+}
```
Comment (juliushaertl) [inline src/shared/components/ncTable/sections/CustomTable.vue:397]: Not sure I fully get those two rules try to achieve, shouldn't we just target something like:

```
:deep(.text-editor__wrapper .paragraph-content:last-child)
```

File: src/shared/components/ncTable/sections/CustomTable.vue (hunk 1 above)
Comment (JuliaKirschenheuter) [reply src/shared/components/ncTable/sections/CustomTable.vue:467]: already removed 🤔

Comment (juliushaertl) [reply src/shared/components/ncTable/sections/CustomTable.vue:467]: Sorry was referring remove the change of removing this line from the commit. It actually causes a conflict now which just would need to be resolved.

File: src/shared/components/ncTable/sections/CustomTable.vue (hunk 2 above)
Comment (JuliaKirschenheuter) [reply src/shared/components/ncTable/sections/CustomTable.vue:397]: thank you! Took your suggestion!

Regarding:

```
:deep(.text-editor__wrapper div.ProseMirror > *:first-child) {
	margin-top: 0!important;
}
```

still think this margin-top have to be removed too
Screenshot from 2024-07-23 14-56-35

Comment (juliushaertl) [reply src/shared/components/ncTable/sections/CustomTable.vue:397]: Ah right, that makes sense to drop then as well of course 👍

**Review by juliushaertl — APPROVED** [review]

**Pull request merged**